doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -6,57 +6,26 @@
 
 ### Features
 
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
+Create a new CLI for iceoryx2 `iox2-config`
 
-* Add `PeriodicTimer` into POSIX building blocks [#425](https://github.com/eclipse-iceoryx/iceoryx2/issues/425)
+`iox2 config` can `show` the configuration of iceoryx currently in use
+or `show` the configuration of the sytem and `generate` a new
+configuration file at the default location iceoryx2 is looking for.
 
-### Bugfixes
-
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
-
-* Example text [#1](https://github.com/eclipse-iceoryx/iceoryx2/issues/1)
+* Add CLI to display complete system configuration [#432](https://github.com/eclipse-iceoryx/iceoryx2/issues/432)
 
 ### Refactoring
 
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
-
-* Example text [#1](https://github.com/eclipse-iceoryx/iceoryx2/issues/1)
-
-### Workflow
-
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
-
-* Example text [#1](https://github.com/eclipse-iceoryx/iceoryx2/issues/1)
-
-### New API features
-
-<!--
-    NOTE: Add new entries sorted by issue number to minimize the possibility of
-    conflicts when merging.
--->
-
-* Example text [#1](https://github.com/eclipse-iceoryx/iceoryx2/issues/1)
+Remove the `print_system_configuration()` function in
+`iceoryx2-bb/posix/src/system_configuration.rs` file and move it into the CLI `iox2-config`
+[#432](https://github.com/eclipse-iceoryx/iceoryx2/issues/432)
 
-### API Breaking Changes
+### New CLI features
 
-1. Example
+```bash
+   iox2-config show system
 
-   ```rust
-   // old
-   let fuu = hello().is_it_me_you_re_looking_for()
+   iox2-config show current
 
-   // new
-   let fuu = hypnotoad().all_glory_to_the_hypnotoad()
-   ```
+   iox2-config generate
```

Comment:
```suggestion
   iox2 config generate
```

Reply:
@orecham why replace iox2-config with iox2 config? For me, iox2-config is the name of the generated binary.

Reply:
@brosier01 The release notes should illustrate the expected usage. The `iox2-config` binary is an implementation detail.